test/003.test.js

```diff
@@ -1,155 +0,0 @@
-/* eslint-disable no-unused-expressions */
-/* eslint-disable no-undef */
-import * as Factory from './lib/factory.js';
-chai.use(chaiAsPromised);
-const expect = chai.expect;
-
-describe('003 protocol operations', () => {
```

Comment:
Where is this file in the new setup?

Reply:
See https://github.com/standardnotes/snjs/blob/migrate-chai-tests/packages/snjs/test/lib/003.encryption.test.ts

Reply:
Old 003.test.js had 155 lines, new file only has 41 lines.

Old: https://github.com/standardnotes/snjs/blob/c1617392ff492d3a0fb68351a57db7f3289eed33/test/003.test.js
New: https://github.com/standardnotes/snjs/blob/migrate-chai-tests/packages/snjs/test/lib/003.encryption.test.ts

Reply:
Fixed 👍🏽